.github/ISSUE_TEMPLATE/__en-US.md

```diff
@@ -9,14 +9,10 @@ about: Report an issue
 2. 
 3.
 
-**Full output of `minikube logs` command:**
-<details>
-
-
-</details>
+**Run `minikube logs --file=logs.txt` and attach the log file to this issue:**
```

Comment:
maybe for the word attach ( we could say how to attach, like Drag the file into this issue)

Reply:
Good idea, done